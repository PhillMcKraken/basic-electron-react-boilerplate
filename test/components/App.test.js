import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import { mount } from 'enzyme';

import App from '../../src/components/App';

describe('App Component', () => {

  it('+++ should compare the component with a snapshot', () => {
    const component = ReactTestRenderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })

  it('+++ should compare the text h1 in portugues.', () => {
    const wrapper = mount(<App />);
    expect(wrapper.find('h1').render().text()).toEqual('Hello, Electron!');
  })

})
