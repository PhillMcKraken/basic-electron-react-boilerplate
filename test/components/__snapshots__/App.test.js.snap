// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component +++ should compare the component with a snapshot 1`] = `
<div>
  <h1>
    Hello, Electron!
  </h1>
  <p>
    I hope you enjoy using basic-electron-react-boilerplate to start your dev off right!
  </p>
</div>
`;
